### ISSUE TITLE
Fix API doc parameters and documentation (Angular specific, and others)

### ISSUE THREAD

**Issue by alex3683** [LaxarJS/laxar-patterns#78]
There are some parameters still containing "scope" (for example `registerFlagFromFeature` in the flag handler). This must be changed into something universal, for example "context".

**Comment by x1B** [comment]
Renamed issue, let's include non-Angular-related API doc cleanup work as well.

**Comment by alex3683** [comment]
- [ ] `@memberOf` => `@memberof`
- [ ] remove `@private` docs

**Comment by alex3683** [comment]
Breaking:
- flags: `scopeKey` => `contextKey`

**Comment by alex3683** [comment]
Implemented on `master` (`v2.0.0`)

### Breaking Change

Since LaxarJS is independent from AngularJS starting with 2.x all occurrences of _scope_ (which referenced the AngularJS widget scope) are renamed to _context_ (generic AxContext instance provided by the widget runtime). The latter is agnostic of any widget implementation technology. In fact for AngularJS the scope extends the AxContext instance.
For the _flags_ library not only the documentation needed to be changed, but also the `scopeKey` option of `registerFlag` and `registerFlagFromFeature`.
So this
```js
flags.handlerFor( context )
   .registerFlagFromFeature( 'superFeature.superAttribute.disableOn', {
        scopeKey: 'flags.disabled',
   } );
```
becomes this
```js
flags.handlerFor( context )
   .registerFlagFromFeature( 'superFeature.superAttribute.disableOn', {
        contextKey: 'flags.disabled',
   } );
```
The same applies to `registerFlag`.
